docs/community/contribution.md

```diff
@@ -68,6 +68,23 @@ If you find a bug in the source code, you can help us by submitting an issue to
 
 Or if you see errors in the documentation you can report them [here][github-docs-issue]
 
+Please do not label issue as a bug - we have to verify if the issue is not a feature, nor the thing that we haven't though of yet.
+
+#### How to make a pull request with a Bug?
```

Comment:
```suggestion
#### How to make a pull request with a Bugfix?
```
hopefully ;P

Reply:
Indeed not a bug.. :D we all know how to make them :D